custom_components/hacs/base.py

```diff
@@ -674,12 +674,12 @@ async def startup_tasks(self, _=None) -> None:
             async_track_time_interval(self.hass, self.async_check_rate_limit, timedelta(minutes=5))
         )
         self.recurring_tasks.append(
-            async_track_time_interval(self.hass, self.async_process_queue, timedelta(minutes=10))
+            async_track_time_interval(self.hass, self.async_process_queue, timedelta(minutes=30))
```

Comment:
Looking at the description, it is probably these 2 you are looking for <https://github.com/hacs/integration/pull/3688/files#diff-49e399d00036b3a319719192cd99a76fd838f6c9f2c01e026b666e605a65de27R640-R651>.
But as you can see, these are already quite high (they were set to that after a meeting with GitHub early last year).

However, these handlers will not be used from the next version (2.0.0), as this integration will use the new data source for these recurring tasks (like <https://data-v2.hacs.xyz/integration/data.json>).

From that version, the integration will only hit the GitHub API when a user triggers it to show the most up-to-date content.
Some more information of that can be found here: https://experimental.hacs.xyz/docs/faq/data_sources